source/includes/_api.md

```diff
@@ -152,6 +152,7 @@ Property | Type      | Description
 `sharedSecret` | *string*  | Shared secret information which will be used to sign the `paymentReceipt` used to [authenticate](#retriving-auth) user during digital goods [retrieval](#retrieving-goods).
 `url`    | *string*  | URL of the web page which contains the good. Used as a reference in the [Dashboard](https://dashboard.satoshipay.io/performance/goods).
 `title`  | *string*  | Title of the good for reference in the provider dashboard.
+`purchaseValidityPeriod`  | *string*  | Time in milliseconds until the good's payment expires..
```

Comment:
This is not quite right. We use the package `ms` to allow for every nice way to format this duration:

https://www.npmjs.com/package/ms

Reply:
please make a suggestion for how to phrase this in this documentation. What is acceptable input?